srcpkgs/lsof/template

```diff
@@ -9,38 +9,67 @@ hostmakedepends="perl which"
 short_desc="LiSt Open Files"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="custom:lsof"
-homepage="https://people.freebsd.org/~abe/"
-distfiles="https://fossies.org/linux/misc/lsof-${version}.tar.gz"
-checksum=a9865eeb581c3abaac7426962ddb112ecfd86a5ae93086eb4581ce100f8fa8f4
+homepage="https://github.com/lsof-org/lsof"
+distfiles="https://github.com/lsof-org/lsof/archive/${version}.tar.gz"
```

Comment:
```suggestion
distfiles="https://github.com/lsof-org/lsof/releases/download/${version}/lsof_${version}.linux.tar.bz2"
```

Reply:
Why?

Reply:
We tend to prefer upstream generated tarballs, if they are provided.

Reply:
@ericonr Why so? Why are upstream-generated tarballs preferred over the source code?

Reply:
Usually because they provide pre-generated files. For example, the PR I linked for `soelim` would also allow us to avoid `groff` in `hostmakedepends` at all.

Reply:
In some cases they are also the signed tarballs.

Reply:
I see you thumbs-upped it but can I interest you in using the upstream tarball? It's simpler to make the change now instead of leaving it for a future version. An added advantage is that if a project moves from github, we can update distfile links without changing checksums.

Alternatively, could you explain why you prefer the github snapshot?

I realize I'm really sticking on this point, but I'm just trying to keep the repository under certain guidelines.

Reply:
Ah, sorry. I just read your previous comments in an explanatory tone rather than a suggestive tone. :sweat_smile: 

I was like, _"Oh, okay. That's why they prefer upstream tarballs"_,  
instead of _"Ah, that's why I should change `distfiles` to the upstream tarball URL. Lemme change it."_

Changed.

Reply:
Oops, forgot to update the sha256sums.

Reply:
Done.

Reply:
No problem, thank you!